tests/apple-exporter/components/test-class-table.php

```diff
@@ -55,6 +55,45 @@ public function setUp() {
 </table>
 HTML;
 
+		$this->html_caption = '<figure class="wp-block-table"><table><thead><tr><th>Column Header 1</th><th>Column Header 2</th></tr></thead><tbody><tr><td>Column Data 1</td><td>Column Data 2</td></tr></tbody><tfoot><tr><td>Column Footer 1</td><td>Column Footer 2</td></tr></tfoot></table><figcaption>Caption</figcaption></figure>';
+	}
+
+	/**
+	 * Tests HTML formatting with captions.
+	 *
+	 * @access public
+	 */
+	public function testCaptions() {
+		$component = new Table(
+			$this->html_caption,
+			null,
+			$this->settings,
+			$this->styles,
+			$this->layouts,
+			null,
+			$this->component_styles
+		);
+
+		// Test.
+		$this->assertEquals(
+			array(
+				'role' => 'container',
+				'components' => array(
+					array(
+						"role" => "htmltable",
```

Comment:
Here and in the other files, these properties and their values should be single-quoted.